### ISSUE TITLE
Add Fribourg et al. (2014)

### ISSUE THREAD

**Issue by JanHasenauer** [Benchmarking-Initiative/Benchmark-Models-PEtab#77]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4012420/

SBML: https://www.ebi.ac.uk/biomodels/BIOMD0000000529#Overview

Estimated complexity of implementation: low

**Comment by JanHasenauer** [comment]
@dweindl @LeonardSchmiester @yannikschaelte The model and the dataset is very interesting, however, we have observables
y(t,u) = x_i(t,u)) / x_i(T,U))
with the time point T \geq t. For some measurements the normalisation is even done with respect to another simulation condition u \neq U.
In my opinion we should get more or less the same of we introduce a scaling constant 
y(t,u) = s_i*x_i(t,u)). The fitting problem is not exactly the same, but as for the case above only information about relative changes would be considered.
What do you think?

**Comment by yannikschaelte** [comment]
I agree that the fitting problem would be different. Since x_i(T,U) is based on the simulations which is not available for (t,u), the scalings s_i would need to estimated. Indeed, this also puts info on a relative scale, so the problem would be similar. If necessary, it should be possible to replicate the exact problem in pyPESTO modifying the AMICI outputs manually. This gets a bit nasty for derivatives though ...

**Comment by LeonardSchmiester** [comment]
Agreed. I think it would be fine to implement it as Jan suggested with scalings and accept that it is not 100% the same optimization problem.

**Comment by dweindl** [comment]
If the goal is not replicating the problem exactly, that works for me too. Should be clearly stated somewhere. Implementing the original problem would require some rather complex(?) PEtab adaptations.

**Comment by JanHasenauer** [comment]
> Should be clearly stated somewhere.

Agreed!

I think these slight adaptations should be fine. Should be like using a slightly different statistical model.
